sdk/tools/gcc_plugin_seh/main.cpp

```diff
@@ -31,23 +32,28 @@ int
 VISIBLE
 plugin_is_GPL_compatible = 1;
 
+struct seh_handler
+{
+    bool is_except;
+    unsigned int line;
+};
+
 struct seh_function
 {
     bool unwind;
     bool except;
     tree asm_header_text;
     tree asm_header;
     size_t count;
+    std::vector<seh_handler> handlers;
 
     seh_function(struct function* fun)
         : unwind(false)
         , except(false)
         , count(0)
     {
         /* Reserve space for our header statement */
-        char buf[256];
-        memset(buf, 0, sizeof(buf));
-        asm_header_text = build_string(sizeof(buf), buf);
+        asm_header_text = build_string(20000, "");
```

Comment:
I wonder whether this "20000" could be made a constexpr or a define somewhere in this file.